### PR TITLE
Img_processing: Add optional visuals

### PR DESCRIPTION
**Pull request by TziporaGoldshmidt**

### CODE REVIEW

File: img_processing/CMakeLists.txt [hunk 1]
```diff
@@ -11,6 +11,12 @@ include_directories(${GTEST_INCLUDE_DIRS})
 
 include_directories(include ../logger ../communication/include ../communication/sockets)
 
+# Check if SHOW_FRAMES flag is set
+option(SHOW_FRAMES "Enable showing frames" OFF)
```
Comment (ChedvaErblich) [inline img_processing/CMakeLists.txt]: Do you want it to be off by default? 
Just checking, go with whatever works best for you.

File: img_processing/src/manager.cpp [hunk 2]
```diff
@@ -160,12 +160,15 @@ int Manager::processing(const Mat &newFrame, bool isTravel)
...
-    int key = cv::waitKey(1);
-    if (key == 27) {
-        return -1;
-    }
-    return 1;
+    #ifdef SHOW_FRAMES
+        drawOutput();
+        imshow("aaa", *currentFrame);
+        int key = cv::waitKey(1);
+        if (key == 27) {
```
Comment (ChedvaErblich) [inline img_processing/src/manager.cpp]: This isn't directly related, but it caught my eye. The value '27' is unclear consider defining it as a variable with a meaningful name or, or, at least, adding a comment to explain it. Please make this change in a separate commit.

**Review by ChedvaErblich — CHANGES_REQUESTED** [review]

File: img_processing/CMakeLists.txt (hunk 1 above)
Comment (TziporaGoldshmidt) [reply img_processing/CMakeLists.txt]: changed to on

File: img_processing/src/manager.cpp (hunk 2 above)
Comment (TziporaGoldshmidt) [reply img_processing/src/manager.cpp]: i added #define ESC 27

**Pull request merged**